skbio/stats/tests/test_composition.py

```diff
@@ -157,20 +157,14 @@ def test_clr(self):
     def test_centralize(self):
         cmat = centralize(_closure(self.data1))
         npt.assert_allclose(cmat,
-                            np.array([[0.22474487, 0.22474487,
-                                       0.55051026],
-                                      [0.41523958, 0.41523958,
-                                       0.16952085]]))
+                            np.array([[0.22474487, 0.22474487, 0.55051026],
+                                      [0.41523958, 0.41523958, 0.16952085]]))
         cmat = centralize(_closure(self.data6))
         npt.assert_allclose(cmat,
-                            np.array([[0.22474487, 0.22474487,
-                                       0.55051026],
-                                      [0.41523958, 0.41523958,
-                                       0.16952085]]))
+                            np.array([[0.22474487, 0.22474487, 0.55051026],
```

Comment:
I had to split up the lines due to pep8 errors, so I may not be able to merge this as is ...